hieradata/node/net-dx.cp.lsst.org.yaml

```diff
@@ -2,6 +2,7 @@
 classes:
   - "profile::core::common"
   - "profile::core::net_dx"
+  - "profile::core::x2go_agent"
```

Comment:
There is something strange going on with this host in that `profile::core::common` should be part of the host's role and not directly included at the node level.

Reply:
yes I agree